### DISCUSSION TITLE
Transcript copyright

### DISCUSSION THREAD

**Discussion by AravindKumar-AI** [jdepoix/youtube-transcript-api#188] [Q&A]
I appreciate this great work which helps us a lot. Just wanted to know whether YouTube video transcripts (auto-generated) are copyrighted material or not?

**Comment by jdepoix** [comment]
Hi @Aravindkumar-Rajendran,
I am not a legal expert so I can't really speak to that. I would **assume** that this depends on how/whether the video you are retrieving the transcripts for is copyrighted. However, if you want to use this in a commercial application and are worried about getting into trouble, you should consult a legal expert!
In case you do consult an expert, I would greatly appreciate if you could share your findings with the community! 😊
